README.md

```diff
@@ -62,7 +62,7 @@ pinned to Keras 2, and will no longer be necessary after TensorFlow 2.16.
 
 ```
 pip install --upgrade keras-cv tensorflow
-pip install keras>=3
+pip install -U keras
```

Comment:
LGTM.  
NIT: We could also use `--upgrade` to be consistent with previous line.  But `-U` also works.

Reply:
done